gossip/storage_test.go

```diff
@@ -62,22 +65,34 @@ func (tp *testStorage) WriteBootstrapInfo(info *gossip.BootstrapInfo) error {
 	return nil
 }
 
+type unresolvedAddrSlice []util.UnresolvedAddr
+
+func (s unresolvedAddrSlice) Len() int {
```

Comment:
Usually I've seen this triplet of functions (at least the short ones) put on a single line. https://golang.org/src/sort/sort.go?s=6381:6400#L252